apps/condo/domains/organization/schema/ResetOrganizationService.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * Generated by `createservice organization.ResetOrganizationService --type mutations`
+ */
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT, INTERNAL_ERROR } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { BankIntegrationAccountContext, BankAccountReport, BankAccount, BankTransaction, BankContractorAccount, BankIntegrationOrganizationContext, BankSyncTask, BankAccountReportTask } = require('@condo/domains/banking/utils/serverSchema')
+const { BillingIntegrationOrganizationContext } = require('@condo/domains/billing/utils/serverSchema')
+const { DV_VERSION_MISMATCH } = require('@condo/domains/common/constants/errors')
+const { loadListByChunks } = require('@condo/domains/common/utils/serverSchema')
+const { B2BAppContext } = require('@condo/domains/miniapp/utils/serverSchema')
+const access = require('@condo/domains/organization/access/ResetOrganizationService')
+const { DELETED_ORGANIZATION_NAME } = require('@condo/domains/organization/constants/common')
+const { Organization, OrganizationLink, OrganizationEmployee } = require('@condo/domains/organization/utils/serverSchema')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const errors = {
```

Comment:
```suggestion
const ERRORS = {
```

Reply:
This is constant, it should be ERRORS, codegen behaviour will change soon. Can you please fix? :)